web-client/README.md

```diff
@@ -0,0 +1,24 @@
+# Nimiq web client
+
+This client is a very light client that only includes the necessary dependencies and constructs
```

Comment:
I think I will re phrase this:
This is a web-client intended to be used in web browsers only (no WASI support).
Although it can be built for other targets, it will panic if it is executed outside a web browser.